src/main/kotlin/christmas/ui/Input.kt

```diff
@@ -0,0 +1,48 @@
+package christmas.ui
+
+object Input {
+
+    fun enterReservationDate(): Int {
+        println(
+            """
+            안녕하세요! 우테코 식당 12월 이벤트 플래너입니다.
+            12월 중 식당 예상 방문 날짜는 언제인가요? (숫자만 입력해 주세요!)
+        """.trimIndent()
+        )
+        try {
+            val day = readLine()!!.toInt()
+            if (day < 0 || day > 31) {
+                throw IllegalArgumentException("존재하지 않는 날짜입니다.")
+            }
+            return day
+        } catch (e: NumberFormatException) {
+            throw IllegalArgumentException("숫자 형식이 아닙니다.")
+        }
```

Comment:
```suggestion
        val day = readln().toIntOrNull() ?: throw IllegalArgumentException("숫자 형식이 아닙니다.")
        if (day < 0 || day > 31) {
            throw IllegalArgumentException("존재하지 않는 날짜입니다.")
        }
        return day
```
Would it be more Kotlin style?

Reply:
i am with you